app/errors/index.js

```diff
@@ -1,3 +1,17 @@
-exports.invalidEndpoint = (req, res) => {
-	res.status(404).send({ status: 404, msg: "Not Found" });
+exports.handleInvalidEndpoint =
+	("*",
+	(req, res) => {
+		res.status(404).send({ status: 404, msg: "Not Found" });
+	});
+
+exports.handleCustomErrors = (err, req, res, next) => {
+	if (err.status && err.msg) {
+		res.status(err.status).send({ msg: err.msg });
+	} else next(err);
 };
+
+exports.handlePsqlErrors = (err, req, res, next) => {
+	if (err.code === "22P02") {
+		res.status(400).send({ msg: "Invalid Input" });
+	}
+};
```

Comment:
Nice, love that you've separated these out into a different file to keep `app.js` tidy 🧹